bin/Cargo.toml

```diff
@@ -10,7 +10,7 @@
 
 [package]
 name = "logdna-agent"
-version = "2.1.8"
+version = "2.1.9-rc1"
```

Comment:
We've decided to stick with beta as the format for prerelease builds, omitting the number on the first beta. So this should be `2.1.9-beta`

Reply:
:+1: